CHANGELOG-3.4.md

```diff
@@ -43,6 +44,8 @@
 - Changed `Phalcon\Crypt::setCipher` so that IV length will be reconfigured during setting the cipher algorithm
 - Changed `Phalcon\Crypt::setCipher` so that method will throw `Phalcon\Crypt\Exception` if a cipher is unavailable
 - Fixed regression ([#13308](https://github.com/phalcon/cphalcon/pull/13308)) for `Phalcon\Debug\Dump::output` to correctly work with detailed mode [#13315](https://github.com/phalcon/cphalcon/issues/13315)
+- Do not throw Exception when superglobal does not exist [#13252](https://github.com/phalcon/cphalcon/issues/13252), [#13254](https://github.com/phalcon/cphalcon/issues/13254), [#12918](https://github.com/phalcon/cphalcon/issues/12918),
+[#13351](https://github.com/phalcon/cphalcon/issues/13351)
```

Comment:
@CameronHall Could you please sort out with change log? There is unrelated entries

Reply:
All done :) It's probably worthwhile asking, is there any way to amend a commit (to keep the git history clean) without any ci? Should I be adding `[ci skip]` to the description if I'm force-pushing an amendment like this?

Does `[ci skip]` skip Appveyor as well?